### PR TITLE
Fixed One-to-one property of logs in EL3, EL5, and EL6 #430

### LINKED ISSUE #430: One-to-one property of logs in EL3, EL5, and EL6
We define this property in [Definition 5.6.7](https://library.tbil.org/2024/precalculus/EL6.html#def-one-to-one-property), and I think it would help to write it out in the notation instead of just words: 

$log_b M = log_b N$ if and only if $M=N$.

Also, I would suggest changing the second half of the definition into an Observation. Something like "Notice this definition allows us to set the arguments (M and N) equal to each other once we get each side expressed as a single logarithm with matching bases."

Also, this seems to be the first time it's defined (in EL6), but it is mentioned in the previous section (EL5) as if it was defined in EL3. See

- [Remark 5.5.1](https://library.tbil.org/2024/precalculus/EL5.html#EL5-3-3) 
- [Activity 5.5.2(b)](https://library.tbil.org/2024/precalculus/EL5.html#investigate-product-property-of-logs-3) 
- [Activity 5.5.3(b)](https://library.tbil.org/2024/precalculus/EL5.html#investigate-quotient-property-of-logs-3) 

We need to clear this up and figure out where we want everything located and referenced.

### PR DESCRIPTION
**Pull request by tdegeorge**
Fixed #430

### CODE REVIEW

File: source/precalculus/source/05-EL/05.ptx [hunk 1]
```diff
@@ -14,14 +14,98 @@
...
+      </statement>
+      <answer>
+        <p>
+          Based on what we have seen in parts (b) and (c), we can conclude that <m>M=N</m>.
+        </p>
+      </answer>
+    </task>
+    </activity>
+
+    <definition xml:id="def-one-to-one-property">
```
Comment (AbbyANoble) [inline source/precalculus/source/05-EL/05.ptx:86]: Do we have a tag for property to use here instead of definition?

Comment (siwelwerd) [reply source/precalculus/source/05-EL/05.ptx:86]: Nope.  Maybe `<fact>` will work?

File: source/precalculus/source/05-EL/05.ptx [hunk 2]
```diff
@@ -14,14 +14,98 @@
...
+      <statement>
+        <p>
+          The <term>one-to-one property of logarithms</term> states that if both sides of an equation can be rewritten as a single logarithm with the same base, then the arguments can be set equal to each other.
```
Comment (AbbyANoble) [inline source/precalculus/source/05-EL/05.ptx:89]: The paragraph here is an if-then statement, but then the part below is an if and only if, so they are sort of saying different things.  I'd suggest changing the paragraph to just set up the parameters for the variables of the if and only if, and then the observation below is what the paragraph said.

```suggestion
         For any values <m>S>0</m> and <m>T>0</m>,  and where <m>b>0</m> and <m>b\neq 1</m>, 
         
         <p>
            <me>\log_bM=\log_bN</me> if and only if <me>M=N.</me>
          </p>
         
        <p> This is called the <term>one-to-one property of logarithms</term></p>
```

File: source/precalculus/source/05-EL/05.ptx [hunk 3]
```diff
@@ -14,14 +14,98 @@
 <introduction>
   In this section, we will explore the properties of logarithms and learn how to manipulate them that will be helpful when we are ready to solve logarithmic equations.
   </introduction>
+
+  <remark>
+      Recall from <xref ref="EL3"/> that we can convert between exponential and logarithmic forms. <me>\log_bx=y</me> is equivalent to <me>b^y=x</me>
```
Comment (AbbyANoble) [inline source/precalculus/source/05-EL/05.ptx:19]:
```suggestion
     <p> Recall from <xref ref="EL3"/> that we can convert between exponential and logarithmic forms. <me>\log_bx=y</me> is equivalent to <me>b^y=x</me>.</p>
```

**Comment by StevenClontz** [comment]
This pull request was not created correctly - it will merge into a feature branch rather than `main` 😭:

image

**Comment by StevenClontz** [comment]
I think this should work now. @AbbyANoble needs to leave an approving review before we can merge.

File: source/precalculus/source/05-EL/03.ptx [hunk 4]
```diff
@@ -200,14 +203,19 @@
...
+        <p>
+          Range: <m>(-\infty, \infty)</m>
         </p>
       </answer>
   </task>
 </activity>
+
 <remark>
-  The powers of 10 function <m>P(t)</m> has an inverse <m>L</m>.  This new function <m>L</m> is called the base 10 logarithm.  But, we could have done a similar procedure with any base, which leads to the following definition.
+  The powers of <m>10</m> function <m>P(t)</m> has an inverse <m>L</m>.  This new function <m>L</m> is called the base <m>10</m> logarithm.  But, we could have done a similar procedure with any base, which leads to the following definition.
```
Comment (AbbyANoble) [inline source/precalculus/source/05-EL/03.ptx:216]:
```suggestion
<p>  The powers of <m>10</m> function <m>P(t)</m> has an inverse <m>L</m>.  This new function <m>L</m> is called the base <m>10</m> logarithm.  But, we could have done a similar procedure with any base, which leads to the following definition. </p>
```

File: source/precalculus/source/05-EL/03.ptx [hunk 5]
```diff
@@ -219,11 +227,11 @@
   </statement>
 </definition>
 <remark>
```
Comment (AbbyANoble) [inline source/precalculus/source/05-EL/03.ptx:229]:
```suggestion
<remark xml:id="remark-log-exp-form">
```

File: source/precalculus/source/05-EL/05.ptx [hunk 6]
```diff
@@ -14,14 +14,104 @@
 <introduction>
   In this section, we will explore the properties of logarithms and learn how to manipulate them that will be helpful when we are ready to solve logarithmic equations.
   </introduction>
+
+  <remark>
+     <p> Recall from <xref ref="EL3"/> that we can convert between exponential and logarithmic forms. <me>\log_bx=y</me> is equivalent to <me>b^y=x</me>.</p>
```
Comment (AbbyANoble) [inline source/precalculus/source/05-EL/05.ptx:19]:
```suggestion
     <p> Recall from <xref ref="remark-log-exp-form"/> that we can convert between exponential and logarithmic forms. <me>\log_bx=y</me> is equivalent to <me>b^y=x</me>.</p>
```

File: source/precalculus/source/05-EL/05.ptx [hunk 7]
```diff
@@ -14,14 +14,104 @@
...
+      </statement>
+      <answer>
+        <p>
+          Based on what we have seen in parts (b) and (c), we can conclude that <m>M=N</m>.
+        </p>
+      </answer>
+    </task>
+    </activity>
+
+    <definition xml:id="def-one-to-one-property">
```
Comment (AbbyANoble) [inline source/precalculus/source/05-EL/05.ptx:86]:
```suggestion
    <fact xml:id="def-one-to-one-property">
```

File: source/precalculus/source/05-EL/05.ptx [hunk 8]
```diff
@@ -14,14 +14,104 @@
...
+            <me>\log_bM=\log_bN</me> if and only if <me>M=N.</me>
+          </p>
+         
+        <p> This is called the <term>one-to-one property of logarithms</term></p>
+          <p>
+            <me>\log_bM=\log_bN</me> if and only if <me>M=N</me>
+          </p>
+        </p>
+      </statement>
+    </definition>
```
Comment (AbbyANoble) [inline source/precalculus/source/05-EL/05.ptx:101]:
```suggestion
    </fact>
```

File: source/precalculus/source/05-EL/05.ptx [hunk 9]
```diff
@@ -14,14 +14,104 @@
...
+
+    <observation>
+      <p>
+        Notice this definition allows us to set the arguments (<m>M</m> and <m>N</m>) equal to each other once we get each side expressed as a single logarithm with matching bases.
```
Comment (AbbyANoble) [inline source/precalculus/source/05-EL/05.ptx:105]:
```suggestion
        Notice this fact will eventually help us solve logarithmic equations. If we have an equation where each side is expressed as a single logarithm with matching bases (such as <m>\log_b M = \log_bN</m>), then it follows that the arguments (<m>M</m> and <m>N</m>) are also equal to each other.
```

File: source/precalculus/source/05-EL/05.ptx [hunk 10]
```diff
@@ -14,14 +14,104 @@
...
+      </p>
+    </observation>
    
   <remark>
-    Recall that <m>\log_b M=\log_b N</m> if and only if <m>M=N</m>. In addition, because exponentials and logarithms are inverses, we also know that <m>\log_b(b^{k})=k</m>.
-    In addition, according to the law of exponents, we know that: <me>x^a \cdot x^b=x^{a+b}</me>
+    Recall that exponentials and logarithms are inverses. We know that <m>\log_b(b^{k})=k</m> and according to the law of exponents, we know that: <me>x^a \cdot x^b=x^{a+b}</me>
```
Comment (AbbyANoble) [inline source/precalculus/source/05-EL/05.ptx:110]:
```suggestion
 <p>   Recall that exponential functions and logarithmic functions are inverses. We know that <m>\log_b(b^{k})=k</m> and according to the law of exponents, we know that: 
 <me>x^a \cdot x^b=x^{a+b}</me>
```

File: source/precalculus/source/05-EL/05.ptx [hunk 11]
```diff
@@ -14,14 +14,104 @@
...
+      </answer>
+    </task>
+    </activity>
+
+    <definition xml:id="def-one-to-one-property">
+      <statement>
+        <p>
+         For any values <m>S>0</m> and <m>T>0</m>,  and where <m>b>0</m> and <m>b\neq 1</m>, 
+         
+         <p>
```
Comment (AbbyANoble) [inline source/precalculus/source/05-EL/05.ptx:91]:
```suggestion
         
```

File: source/precalculus/source/05-EL/05.ptx [hunk 12]
```diff
@@ -14,14 +14,104 @@
...
+            <me>\log_bM=\log_bN</me> if and only if <me>M=N.</me>
+          </p>
+         
+        <p> This is called the <term>one-to-one property of logarithms</term></p>
```
Comment (AbbyANoble) [inline source/precalculus/source/05-EL/05.ptx:95]:
```suggestion
        <p> This is called the <term>one-to-one property of logarithms</term>.</p>
```

File: source/precalculus/source/05-EL/05.ptx [hunk 13]
```diff
@@ -14,14 +14,104 @@
...
+          <p>
+            <me>\log_bM=\log_bN</me> if and only if <me>M=N</me>
+          </p>
+        </p>
```
Comment (AbbyANoble) [inline source/precalculus/source/05-EL/05.ptx:99]:
```suggestion

```

File: source/precalculus/source/02-FN/05.ptx
Comment (AbbyANoble) [inline source/precalculus/source/02-FN/05.ptx:1]: I'm not sure why this file is in this pull request? @siwelwerd or @StevenClontz?

Comment (siwelwerd) [reply source/precalculus/source/02-FN/05.ptx:1]: I think Tonya accidentally branched off of a branch, but I think it's fixed now

**Comment by siwelwerd** [comment]
I merged all of your commits @AbbyANoble , but I think it's still a little hard to follow since the remarks aren't showing up right.  Maybe @StevenClontz  can merge #514 into main to fix those, and then merge main into this so it's easier to see.

**Comment by StevenClontz** [comment]
Try that...

**Comment by siwelwerd** [comment]
@AbbyANoble Please review latest preview build, and see if there is any remaining comment from #430 discussion that needs to be addressed here.

**Comment by siwelwerd** [comment]
@AbbyANoble once this preview builds, it should show all of the work with this branch along with Jordan's revised remark from #523

**Comment by siwelwerd** [comment]
> This part of Jordan's revision doesn't show up in this preview.

Okay it's back there now.  I must have hit a wrong button.
